docs/UserGuide.md

```diff
@@ -49,6 +51,14 @@ With this guide, you'll be ready to use GradPad in no time.
 
    * **`exit`** : Exits the app.
 
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+Detailed explanation of each command can be found [here](#commands).
+
+</div>
+
 1. Refer to the [Feature Summary List](#feature-summary-list) below for a summary of all commands.
```

Comment:
Should we put this in the note too? We also need to remove the numbering
```suggestion
Refer to the [Feature Summary List](#feature-summary-list) below for a summary of all commands.
```

Reply:
Oh right, good point HAHAH. Removed the numbering and placed in the note.